apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -147,9 +154,27 @@ function HomeScreenInner({ account }: { account: Account }) {
 
   const contactsAccess = useContactsPermission();
 
+  const refreshAnimatedStyle = useAnimatedStyle(() => ({
+    opacity: refreshOpacity.value,
+  }));
+
+  const forceRefresh = () => {
+    if (!refreshing) {
+      refreshOpacity.value = 0;
+      setShowRetryingTitle(true);
+      onRefresh();
+      refreshOpacity.value = withDelay(300, withTiming(1));
+    }
+  };
```

Comment:
I would recommend simplifying.

- The interaction between `forceRefresh` and `onRefresh` seems complicated.
- Tapping the header doesn't have to animate the whole page downward (as if it was pull-to-refreshed)
- Currently, tapping the `<OfflineHeader/>` on any *other* page (ie, not HomeScreen) doesn't refresh...